examples/getting-started-movielens/04-Triton-Inference-with-TF.ipynb

```diff
@@ -35,26 +35,7 @@
     "\n",
```

Comment:
<p>is this transformed dataset? if you are only serving TF model not NVT + TF then we should send the transformed dataset as a request, not the raw. Can you add clarification please saying that we send transformed dataset (if this is the case) due to such such reason? if this is not transformed dataset, then this is not accurate, if you are only serving the model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>yes, this is the transformed dataset, the one our model was trained on, will add clarification</p><p> </p>